bin/build-release

```diff
@@ -179,6 +179,17 @@ ln -snf buildroot-${BR_VER} buildroot
 ( cd buildroot && QUILT_PATCHES="$PWD/../patches-buildroot" quilt push -a )
 
 echo "$VER" > "src/etc/cirros/version"
+cat << EOF > "src/etc/os-release"
+PRETTY_NAME="CirrOS $VER"
+NAME="CirrOS"
+VERSION_ID="$VER"
+ID=cirros
+HOME_URL="https://github.com/cirros-dev/cirros"
+BUG_REPORT_URL="https://github.com/cirros-dev/cirros/issues"
+BUILDROOT_VERSION="$BR_VER"
+KERNEL_VERSION="$KVER"
+GRUB_VERSION="$GVER"
```

Comment:
lets drop BUILDROOT_VERSION, KERNEL_VERSION and GRUB_VERSION.

As for my reasoning:
 * They're not standard per a quick google result [os-release(5)](https://manpages.ubuntu.com/manpages/jammy/en/man5/os-release.5.html)
 * I don't have a use-case that they solve or help solve.  What would they be used for?
 * Ubuntu/RHEL/... don't advertise versions of their components in this file.

Reply:
sgtm. I originally added those variables for extra distro-level metadata (potentially for the curious), but I do agree that there does not seem to be an obvious use-case for this info + it is unstandard information. I will drop this information for now and we can add it back if requested by popular demand.